src/ol/source/stamen.js

```diff
@@ -35,7 +35,8 @@ ol.source.Stamen = function(options) {
     opaque: layerConfig.opaque,
     reprojectionErrorThreshold: options.reprojectionErrorThreshold,
     tileLoadFunction: options.tileLoadFunction,
-    url: url
+    url: url,
+    wrapX: options.wrapX
```

Comment:
The default for wrapX in `ol.surce.XYZ` is `true`, in this case if omitted in 'options' the default will be `false`, this could be a better solution:
```
wrapX: typeof(options.wrapX) !== 'undefined' ? options.wrapX : true
```